site-manage/site-group-manager/src/main/resources/Messages.properties

```diff
@@ -26,7 +26,7 @@ index.table.assignments=Assignments
 index.table.joinableset=Joinable set
 index.table.members=members
 index.table.selectallnone=Select All/None
-index.warning.groupempty=There are no groups defined.
+index.warning.groupempty=There are currently no custom groups defined. You can create new groups using one of the options above."
```

Comment:
Can you remove the trailing double quote here?

Reply:
Done, sorry about that!